.azure-pipelines/run-test-template.yml

```diff
@@ -56,9 +56,14 @@ steps:
 
 - script: |
     pwd
-
+    git_user_name=$GIT_USER_NAME
+    git_api_token=$GIT_API_TOKEN
+    
     parent_dir=$(basename $PWD)
-    docker exec sonic-mgmt-2 bash -c "/var/src/$parent_dir/tests/kvmtest.sh -en -T ${{ parameters.tbtype }} -d /var/src/$parent_dir ${{ parameters.tbname }} ${{ parameters.dut }} ${{ parameters.section }}"
+    docker exec -e GIT_USER_NAME=$(git_user_name) -e GIT_API_TOKEN=$(git_api_token) sonic-mgmt-2 bash -c "/var/src/$parent_dir/tests/kvmtest.sh -en -T ${{ parameters.tbtype }} -d /var/src/$parent_dir ${{ parameters.tbname }} ${{ parameters.dut }} ${{ parameters.section }}"
```

Comment:
Update this to:
```
    docker exec -e GIT_USER_NAME=$GIT_USER_NAME -e GIT_API_TOKEN=$GIT_API_TOKEN sonic-mgmt-2 bash -c "/var/src/$parent_dir/tests/kvmtest.sh -en -T ${{ parameters.tbtype }} -d /var/src/$parent_dir ${{ parameters.tbname }} ${{ parameters.dut }} ${{ parameters.section }}"
```
And line 59, 60 can be deleted.

Reply:
done